client/me/purchases/manage-purchase/index.jsx

```diff
@@ -516,6 +517,26 @@ class ManagePurchase extends Component {
 				'Transfers an existing domain from another provider to WordPress.com, ' +
 					'helping you manage your site and domain in one place.'
 			);
+		} else if ( isGSuiteOrGoogleWorkspace( purchase ) ) {
+			description = translate( 'Account for %(domain)s', {
+				args: {
+					domain: purchase.meta,
+				},
+			} );
+
+			if ( purchase.purchaseRenewalQuantity ) {
+				description = translate(
+					'%(numberOfUsers)d user for %(domain)s',
+					'%(numberOfUsers)d user for %(domain)s',
```

Comment:
The second one needs to be in plural: users :)

Reply:
Oh right, good catch!